galerautils/src/gu_dbug.c

```diff
@@ -1532,7 +1532,7 @@ GU_DBUGOpenFile(const char *name, int append)
 
     if (name != NULL) {
 	strcpy(_gu_db_stack->name, name);
-	if (strcmp(name, "-") == 0) {
+	if (strlen(name) == 1 && name[0] == '=') {
```

Comment:
What is the reason for this change? Some micro optimization in debug code?

Besides, string length check is unnecessary and there is a typo '-' changes to '='.